shared/src/utils/stats/SurveyStatsCalculator.ts

```
@@ -0,0 +1,131 @@
+
+	private static aggregateSurveyData(surveysData: Survey[]): {
+		typeCounts: { [type: string]: number };
+		aggregatedData: { [key: string]: { [key: string]: SurveyAnswersByType } };
+		oldestDate: Date;
+	} {
+		const aggregatedData: { [key: string]: { [key: string]: SurveyAnswersByType } } = {};
+		const typeCounts: { [type: string]: number } = {};
+		let oldestDate = new Date();
+
+		surveysData.forEach((survey) => {
+			if (this.isCompletedSurvey(survey)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)